enviro/__init__.py

```diff
@@ -172,6 +172,8 @@ def connect_to_wifi():
 
   logging.info("  - ip address: ", ip)
   """
+  import rp2
+  rp2.country("GB") 
```

Comment:
This should probably be a config setting. It may create wifi connection issues in other countries.

Reply:
This would have been a good thing for me to think of before merging the pull request 🤦. Any chance you can raise an issue for it, or perhaps a PR yourself if have the time?

Reply:
Sure, I may have some time this weekend to create a pull request to use a config setting. For now do you think just removing these two lines will be enough?

Reply:
You can remove them on your local copy if they're causing issues. There's no urgency to remove them from this repository though, as most people download versions from the releases tab.